### PR TITLE
I have no fucking idea what I'm doing someone else fix this properly

### PR DESCRIPTION
**Pull request by silicons**
Reverts tgstation/tgstation#22069
Thanks Cobby.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Elaborate

**Comment by Grazyn** [comment]
There's currently a bug that makes you hit (with hit animation) everything, syringes hitting beakers when drawing something, hitting your own backpack etc.

**Comment by Jazaen** [comment]
Do merge this once the time is up, it's driving me insane.

**Comment by Supermichael777** [comment]
Revert the dumb buggy grab intent change. it fucking broke everything

File: code/_onclick/click.dm [hunk 1]
```diff
@@ -234,8 +234,6 @@
 		var/mob/living/carbon/human/H = user
 		H.dna.species.grab(H, src, H.martial_art)
 		H.next_click = world.time + CLICK_CD_MELEE
-	else
```
Comment (Supermichael777) [inline code/_onclick/click.dm]: swap this with an indented return and ...()

**Comment by Supermichael777** [comment]
This shouldn't be causing that are you on harm intent? if your on harm intent you try and smash objects with eachother

**Comment by FantasticFwoosh** [comment]
This wouldn't be a problem without the damage rework making stuff break permanently in some cases.

**Comment by ChangelingRain** [comment]
This will break things that aren't human trying to pull humans.

**Comment by KorPhaeron** [comment]
What bug was cobby originally fixing. Can we go back to that?

**Comment by Cyberboss** [comment]
I blame the original aggressive grab pr. I literally never saw it happen until someone brought it up here

**Comment by kevinz000** [comment]
@KorPhaeron I'm loading shells into a shotgun. What used to be an instant action is now an action impeded by melee clickdelay and me hitting the shotgun with the shell making a smashing sound.

**Comment by kevinz000** [comment]
Yeah no I don't think this is the sole cause and I'd rather not fuck more things up by trying to fix it .

**Pull request closed without merging**